src/window.jl

```diff
@@ -106,6 +116,15 @@ body(win::Window, html::String) =
 const initcss = """
   <style>html,body{margin:0;padding:0;border:0;text-align:center;}</style>
   """
+body(win::Window, x) =
+  # FIXME: use tohtml
+  body(win, stringmime(MIME"text/html"(), x))
+
+head(win::Window, html::String) =
+  @js win document.head.innerHTML = document.head.innerHTML + $html
+
+head!(win::Window, html::String) =
+  @js win document.head.innerHTML = $html
```

Comment:
I think it would be best to swap the `!` here, since the first method is the one that actually modifies the head. Right now it's a bit inconsistent